packages/vite/src/node/plugins/css.ts

```diff
@@ -360,12 +366,14 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
 
       let code: string
       if (usedRE.test(id)) {
-        if (inlined) {
-          code = `export default ${JSON.stringify(
-            await minifyCSS(css, config)
-          )}`
+        if (modulesCode) {
+          code = modulesCode
         } else {
-          code = modulesCode || `export default ${JSON.stringify(css)}`
+          let content = css
+          if (inlined) {
```

Comment:
I left this condition as same as it was. But is this condition intended?
I feel it natural to change this condition to `!!config.build.minify` which will fix #5471.

Reply:
I think this may be an artifact of the previous refactorings. I say we better change it to `config.build.minify` in this PR